lib/testscript_engine/testscript_runnable.rb

```diff
@@ -40,11 +40,13 @@ def autodelete_ids
     @autodelete_ids ||= []
   end
 
-  def initialize(script, block, options, available_profiles = nil)
+  def initialize(script, block, options, engine = nil, available_profiles = {}, bound_variables = {})
     self.get_fixture_block = block
     self.options = options
     self.client_util = FHIR::Client.new('')
     self.available_profiles = available_profiles
+    self.engine = engine
```

Comment:
I think by having engine object part of runnable initialization has pros and cons and we may make it a bit clearer or simpler as to what information the engine handles vs the runnable handles. But that is out of the scope in the last hour of this project. For now I'm okay to this change if you'd prefer that way.

Reply:
agreed - this was a quick solution that could be handled better in another way.  Main need is to pass the report back to the engine, which could be done in another way.